packages/big-design/src/components/Flex/withFlex.ts

```diff
@@ -6,7 +6,10 @@ import { FlexedItemProps, FlexedOverload, FlexedProps } from './types';
 export const withFlexedContainer = () => css<FlexedProps>`
   ${({ alignContent, theme }) => alignContent && getFlexedStyles(alignContent, theme, 'align-content')};
   ${({ alignItems, theme }) => alignItems && getFlexedStyles(alignItems, theme, 'align-items')};
+  ${({ flexColumnGap, theme }) => flexColumnGap && getFlexedStyles(flexColumnGap, theme, 'column-gap')};
   ${({ flexDirection, theme }) => flexDirection && getFlexedStyles(flexDirection, theme, 'flex-direction')};
+  ${({ flexGap, theme }) => flexGap && getFlexedStyles(flexGap, theme, 'gap')};
```

Comment:
What happens if we have `gap`, `column-gap`, & `row-gap` props in the same element? Does `gap` supersede the others or does it still depends on the cascade? Wondering if we should move `gap` before both `column-gap` and `row-gap`.

Reply:
ohh yeah good catch, thanks!  I think it's gonna overrides. I agree, the gap should be before column and row 👍🏼